site/layouts/partials/scripts.html

```diff
@@ -4,79 +4,29 @@
   <script src="/docs/{{ .Site.Params.docs_version }}/dist/js/boosted.bundle.js"></script>
 {{- end }}
 
-<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
+{{- $esbuildOptions := dict "target" "es2019" -}}
+{{- $targetDocsJSPath := path.Join "/docs" .Site.Params.docs_version -}}
+{{- $esbuildOptions = merge $esbuildOptions (dict "minify" "true") -}}
```

Comment:
TODO: double-check this part because our Boosted code wasn't within a `{{- if hugo.IsProduction -}}` because of tarteaucitron.